packages/editor/src/components/alignment-toolbar/index.js

```diff
@@ -22,13 +28,18 @@ const ALIGNMENT_CONTROLS = [
 	},
 ];
 
-export default function AlignmentToolbar( { value, onChange } ) {
+function AlignmentToolbar( { isCollapsed, isLargeViewport, value, onChange } ) {
 	function applyOrUnset( align ) {
 		return () => onChange( value === align ? undefined : align );
 	}
 
+	const activeAlignment = find( ALIGNMENT_CONTROLS, ( control ) => control.align === value );
+
 	return (
 		<Toolbar
+			isCollapsed={ isCollapsed || ! isLargeViewport }
```

Comment:
I think we may need an option to disable this forced collapsing on mobile. On the heading block the AlignmentToolbar component is used on the sidebar and collapsing it makes the result feel strange.
![sep-07-2018 19-13-29-1](https://user-images.githubusercontent.com/11271197/45236392-a9b73580-b2d3-11e8-9d25-8ff32de40448.gif)

If the resolution is really small it becomes even impossible to change the alignment, that seems unexpected and I'm not certain of the reason why it happens:
![sep-07-2018 19-14-03-2](https://user-images.githubusercontent.com/11271197/45236462-d3705c80-b2d3-11e8-9f44-0c2046957d05.gif)